src/components/Element/Article.jsx

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+import clsx from 'clsx';
+import { useState, useEffect, useRef } from 'react';
+import { Calendar, Tip, Content } from './ArticleElement';
+
+export function Article() {
+  const [article, setArticle] = useState([]);
+  const [sentence, setSentence] = useState({});
+  const [position, setPosition] = useState({});
+  const [link, setLink] = useState('');
+  const markRef = useRef(null);
+
+  useEffect(() => {
+    axios
+      .get(`${process.env.REACT_APP_URL}/api/article`)
+      .then((res) => {
+        setArticle(
+          res.data.articles.reduce((acc, curr) =>
+            curr.content.length > acc.content.length ? curr : acc
+          )
+        );
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
+
+  useEffect(() => {
+    if (!sentence) return;
```

Comment:
could this not be written as if (sentence){  ... }. instead of of returning nothing if !sentence  ?